WalkingChallenge/Notifications/Notifications.swift

```diff
@@ -56,17 +56,31 @@ class Notifications: UIViewController {
     }
 
     self.tableView.dataSource = self
+    self.tableView.delegate = self
+    self.tableView.register(NotificationTableViewCell.self, forCellReuseIdentifier: "cell")
     self.tableView.tableFooterView = UIView()
   }
 }
 
 extension Notifications: UITableViewDataSource {
   func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-    return 1
+    //TODO: update with data
+    return 3
```

Comment:
What is the 3 here for?

Reply:
just wanted to see more rows than 1 lol...i guess i'll switch it to 1

Reply:
Well, it all comes down to the modelling of the table.  If you are just going to create 1 section, then this should be the entire count of notifications.  However, if you are going to treat each cell as its own "section" (without a section separator), then this should always be 1.